pex/expose_scripts.py

```diff
@@ -0,0 +1,70 @@
+import hashlib
+import itertools
+import os
+
+from pathlib import Path
```

Comment:
Pex still supports Python 2.7 so you need to astick to those APIs - this will fail the 2.7 CI shards as-is.